contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/util/SimpleHttp.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.drill.exec.store.http.HttpStoragePluginConfig;
 import org.apache.drill.exec.store.http.HttpSubScan;
 import org.apache.drill.exec.store.security.UsernamePasswordCredentials;
+import org.apache.parquet.Strings;
```

Comment:
Please use `StringUtils` from commons-lang3 instead of using this method from the parquet library...

Reply:
@vvysotskyi Thanks for the quick review!  I updated with `StringUtils`.